node/testing/src/scenarios/cluster_runner.rs

```diff
@@ -117,15 +118,15 @@ impl<'a> ClusterRunner<'a> {
         )
     }
 
-    pub fn get_chain_id(&self) -> Option<String> {
+    pub fn get_chain_id(&self) -> Option<ChainId> {
```

Comment:
Do we still need chain id inside cluster?

Reply:
Not sure why we had it in the first place. If in test we are interacting with the ocaml node, we will get the chain_id from there, otherwise chainid didnt matter.